src/common/meta/src/key/datanode_table.rs

```diff
@@ -209,6 +224,7 @@ impl DatanodeTableManager {
                 opts.push(TxnOp::Delete(raw_key))
             }
         }
+        // TODO(niebayes): maybe also check any update on region_wal_options.
```

Comment:
Let's do it right now, it's an implicit bug in the future.

Reply:
Currently, the check for any update on `region_options` is redundant, since the `region_info.region_options` and `new_region_options` must be identical.
See: https://github.com/GreptimeTeam/greptimedb/blob/6b8dbcfb5450efb244eb11ca884cab46a6324d09/src/meta-srv/src/procedure/region_failover/update_metadata.rs#L100-L113

Reply:
Added check for update on region_wal_options.